app.js

```diff
@@ -1,14 +1,18 @@
 const express = require('express');
 const { getTopics } = require('./controllers/controller');
 const { handleCustomErrors, handleServerErrors } = require('./errors');
+const endpoints = require('./endpoints.json');
 const app = express();
 
+app.get('/api', (req, res) => {
```

Comment:
Data shouldn't be imported into the app, so we need to extract this callback function as a controller 👀
If the data is imported into the app, it exists on the memory of the server all the time, even if a request isn't made to the endpoint that responds with it. This is inefficient for the performance of our server.